src/ui/public/modals/confirm_modal.html

```diff
@@ -1,4 +1,16 @@
 <div class="kuiModal" style="width: 450px" data-test-subj="confirmModal">
+  <div class="kuiModalHeader" ng-if="title">
```

Comment:
Do we need this `ng-if` here? It seems unexpected to hide the close button if no `title` is provided. I think we can remove it entirely and just show an empty title element if no `title` is provided.